proxy/src/main/java/com/velocitypowered/proxy/bossbar/VelocityBossBarManager.java

```diff
@@ -0,0 +1,43 @@
+package com.velocitypowered.proxy.bossbar;
+
+import com.google.common.collect.ImmutableList;
+import com.velocitypowered.api.bossbar.BossBar;
+import com.velocitypowered.api.bossbar.BossBarColor;
+import com.velocitypowered.api.bossbar.BossBarManager;
+import com.velocitypowered.api.bossbar.BossBarOverlay;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+import net.kyori.text.Component;
+import org.checkerframework.checker.nullness.qual.NonNull;
+
+public class VelocityBossBarManager implements BossBarManager {
```

Comment:
Actually, now that I think about it, we don't really need a `VelocityBossBarManager` class. Just expose a method on `VelocityServer` to create a boss bar.

Reply:
Sure